go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/prometheus/prometheus v1.8.2-0.20190924101040-52e0504f83ea
 	github.com/rs/cors v1.6.0
 	github.com/securego/gosec v0.0.0-20200316084457-7da9f46445fd
-	github.com/shirou/gopsutil v0.0.0-20190901111213-e4ec7b275ada
+	github.com/shirou/gopsutil v2.20.4+incompatible
```

Comment:
I'm not sure of any reason to prefer using `v0.0.0` format instead of `v2.x.y+incompatible`, so I just updated the regular way via `go get`